src/SME.SGP.Aplicacao/Commands/Armazenamento/SalvarDocumento/SalvarDocumentoCommandHandler.cs

```diff
@@ -26,6 +27,15 @@ public async Task<bool> Handle(SalvarDocumentoCommand request, CancellationToken
             if (existeArquivo)
                 throw new NegocioException("Este usuário já possui um arquivo");
 
+            var usuario = await mediator.Send(new ObterUsuarioPorIdQuery(request.SalvarDocumentoDto.UsuarioId));
+            var tiposDocumentos = await mediator.Send(new ObterTipoDocumentoClassificacaoQuery());
+
+            var classificacao = tiposDocumentos.FirstOrDefault(t => t.Id == request.SalvarDocumentoDto.TipoDocumentoId).Classificacoes.FirstOrDefault(c => c.Id == request.SalvarDocumentoDto.ClassificacaoId);
+            
+            if(!usuario.Perfis.Where(u => u.NomePerfil == classificacao.Classificacao).Any())
```

Comment:
acredito que essa validação só possa ocorrer quando o tipo de documento é "Plano de Trabalho", quando o tipo for "Documento" não tenho certeza se o bloqueio se aplica. Podes confirmar com o Marlon ?